src/modules/networkcfg/main.py

```diff
@@ -53,4 +53,13 @@ def run():
             except FileExistsError:
                 pass
 
+    # We need to overwrite the default resolv.conf in the chroot.
+    source_resolv = "/etc/resolv.conf"
+    target_resolv = os.path.join(root_mount_point, "etc/resolv.conf")
+    if os.path.exists(source_resolv):
```

Comment:
Most distros I have seen ship an empty file with a few comments, so a check like that would not work.

Shipping a "default" DNS configuration with the root image seems weird. A lot of people set their own DNS anyway, especially if they are on a LAN or similar networks. In short, I think whatever the user has configured on the live system is good as a source of truth, since it is their responsibility to ensure it works at that point, not the distro maintainer's.

If we want to cover the uncommon case of default resolv.conf in the root image we can add a "don't overwrite resolv.conf" parameter in the module settings.